tests/p2p/test_get_best_blockchain.py

```diff
@@ -281,8 +278,7 @@ def test_best_blockchain_from_storage(self):
         self.assertTrue(block is memo_block)
 
         # cache miss if best block doesn't match
-        fake_block = HeightInfo(1, 'fake hash')
-        manager1._latest_n_height_tips = [fake_block]
```

Comment:
Why have you removed this line?

Reply:
After this test was typed, mypy complained that `HathorManager` does not have a `_latest_n_height_tips` property. Therefore, this was useless (probably an artifact from some older code) and removing it doesn't break the test.

Reply:
I noticed that this property was missing from the `TransactionStorage` typing, so it was either refactored way from `HathorManager` and  this test is outdated or this test has always been wrong.

Would it be better to comment this lines instead and add a `FIXME/XXX` note to review the test?

Also the comment above the 2 lines that are being removed looks like it refers to the `_latest_n_height_tips` so at the very least if we remove the line we should remove the comment.

Reply:
Ok, I commented the line and added a `FIXME` in https://github.com/HathorNetwork/hathor-core/pull/973/commits/20a973dcba804e43868c583e811af6c878a21f22

Reply:
Thanks!